LL/dal_corpri_arad.ttl

```diff
@@ -5321,9 +5321,14 @@
     rel:childOf <#NadBuidb-fb5b07ec>.
 
 <#NadBuidb-fb5b07ec>
-    owl:sameAs <#NadBuidb>;
+    a foaf:Person;
+    irishRel:nomName "Nad Buidb";
     rel:childOf <#Illaind>.
 
+# This Nad Buidb was marked as owl:sameAs <#NadBuidb> but I don't see
+# any reason for thinking so. The text presents them in close proximity
+# with different fathers. EPT
+
```

Comment:
I shouldn't have been so hasty. Re-reading the passage, I can see how both interpretations are possible. Here is the passage, with my interpretation of what is going on in square brackets: 

> 
> 
> It é tra mc Dunlaing inso. **Illand**. Ailill. Eochaid. Fergus. **Nad Buidb** **[=Nad Buidb1]**. Dubthach Lethderg. Mundech nó Oengus Dilmend.
> 
> Is h-í Cland **Illaind** mc Dunlainge in so i .h. Maele Dúin & sil Aeda Guaire .&c.
> 
> Cland Bruidgi mc Nad Buidb**[=Nad Buidb2]** .h. Blaithmeic & h. Fhingin ó Alind .h. Thectais & sil Setnai Móir .i. h. Buide. Et síl m-Bruidgi eter Thulcaind & Abaind. Foithri. dia tá Aed Cluana Mór .i. Aed mc Eogain mc Bruidge mc Nad Buidb**[=Nad Buidb2]** .m. **Illaind** mc Dunlainge. 

There is no reason why Illand, brother of Nad Buidb1, should not have a son callled Nad Buidb (Nad Buidb2). But "Cland Bruidgi mc Nad Buidb", mentioned after only Nad Buidb1 has been mentioned, does imply that they are thought to be the same person. 

We might here have an instance of poor-quality doctoring of the genealogy. It might have been in someone's interests at some point either to promote the idea that the descendants of Nad Buidb had no independent claims within the kin-group, as Nad Buidb was but a son of Illand. Or the reverse might have happened: the descendants of Nad Buidb declared their independence by making him a brother, not a son, of Illand. In either case, only half the doctoring of this text was actually accomplished. 

That doesn't help us work out whether the two Nad Buidbs mentioned are to be identified with each other. Sorry. What do you think?